### PR TITLE
adding pt-br to language

### PR DESCRIPTION
**Pull request by sombriks**
there is an issue with golang > 1.17, but i didn't fixed it in this pull request. 

please add portuguese as one of the supported languages

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:latest
+FROM golang:1.17
```
Comment (imeoer) [inline Dockerfile:1]: Maybe use `golang:1.18` here, keep it consistent with `go.mod`.

File: README.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ ink convert /path/_posts
 
 ### Building from source
 
-Local Build
+Local Build ([for go 1.17](https://go.dev/doc/go-get-install-deprecation)
```
Comment (imeoer) [inline README.md:99]: Ditto.

**Review by imeoer — COMMENTED** [review]
Thanks for the contribution!

**Comment by w568w** [comment]
> there is an issue with golang > 1.17

Just out of curiosity, what is wrong with the latest Go?

**Comment by sombriks** [comment]
>> there is an issue with golang > 1.17

> Just out of curiosity, what is wrong with the latest Go?

By trying the `go get github.com/InkProject/ink` command under **go1.19.5 linux/amd64** or more recent it gives the following error:

image

I think it could be into another pull request, since the issue with using go install instead is the deep change on that documentation part

File: Dockerfile (hunk 1 above)
Comment (sombriks) [reply Dockerfile:1]: sure!

**Comment by w568w** [comment]
> By trying the `go get github.com/InkProject/ink` command under **go1.19.5 linux/amd64** or more recent it gives the following error

#145 fixes this now. @imeoer do you think it could be merged now?

**Comment by imeoer** [comment]
@w568w It's ok for me, thanks @sombriks !

**Pull request merged**